src/patcher.js

```diff
@@ -6,6 +6,19 @@ const { BrowserWindow, app, session } = electron;
 const electronPath = require.resolve('electron');
 const discordPath = join(dirname(require.main.filename), '..', 'app.asar');
 
+console.log('Patching')
```

Comment:
is this intentional? nothing inherently wrong with it, just making sure

Reply:
Non-intentional, removed with everything else